frontend/src/routes/Home/Search/components/SavedSearchQueries.tsx

```diff
@@ -164,8 +166,8 @@ export default function SavedSearchQueries(props: {
                                 key={index}
                                 cardHeader={{
                                     hasIcon: true,
-                                    title: query.name,
-                                    description: query.description,
+                                    title: t(query.name),
```

Comment:
Are these predefined searches, or are they user supplied? Generally we should only provide literal strings to the `t` function; otherwise, all possible calls to the `t` function should be enumerated in a comment. Same comment for line below.

Reply:
I handled this elsewhere for the suggestedQueryTemplates, so removing here.